plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -386,6 +393,9 @@ fu_synaptics_rmi_device_setup (FuDevice *device, GError **error)
 	/* set page */
 	if (!fu_synaptics_rmi_device_set_page (self, 0, error))
 		return FALSE;
+
+	/* Force entering iep mode again. */
+	fu_synaptics_rmi_device_set_iepmode (self, FALSE);
 	if (!fu_synaptics_rmi_device_enter_iep_mode (self, error))
```

Comment:
why doesn't the call for `fu_synaptics_rmi_device_enter_iep_mode` carry the void call for `fu_synaptics_rmi_device_set_iepmode`?

Reply:
It will determine whether it needs issuing entering iep mode command really to device depends on in_iep_mode in fu_synaptics_rmi_device_enter_iep_mode(), and device will be back to non-iep mode by itself in some cases without any notification in application layer. I put fu_synaptics_rmi_device_set_iepmode() to make in_iep_mode be FALSE to let entering iep mode command be issued.